apps/src/sites/studio/pages/schoolInfo.js

```diff
@@ -188,36 +188,45 @@ window.SchoolInfoManager = function (existingOptions) {
     return ['public', 'charter'].indexOf($('#school-type').val()) > -1;
   }
 
+  function isHomeschool() {
+    return $('#school-type').val() === 'homeschool';
+  }
+
+  function isAfterSchool() {
+    return $('#school-type').val() === 'afterschool';
+  }
+
   function isUs() {
     return $('#school-country').val() === 'US';
   }
 
+  function closestFormGroupOrItemBlock(selector) {
+    var closest = $(selector).closest('.form-group');
+    if (closest.length === 0) {
+      closest = $(selector).closest('.itemblock');
+    }
+    return closest;
+  }
+
   function show(selector) {
-    $(selector).closest('.form-group').show();
+    closestFormGroupOrItemBlock(selector).show();
   }
 
   function clearAndHide(selector) {
     $(selector).val('');
-    $(selector).closest('.form-group').hide();
+    closestFormGroupOrItemBlock(selector).hide();
   }
 
   $('#school-country').change(function () {
     $("input[type=submit]").prop("disabled", false);
     if ($(this).val() === 'US') {
       clearAndHide('#school-name');
       clearAndHide('#school-address');
-      // Show fields corresponding to the current contents of the type dropdown.
-      $('#school-type').change();
-    } else { // non-US
-      clearAndHide('#school-zipcode');
-      clearAndHide('#school-state');
-      clearAndHideDistrict();
-      clearAndHideSchool();
-      show('#school-name');
-      show('#school-address');
     }
+    $('#school-type').change();
```

Comment:
Because we're adding more non-US behavior, it makes more sense to handle in the #school-type change method